packages/css/src/components/ListItem.css

```diff
@@ -0,0 +1,81 @@
+.ListItem {
+  padding: var(--space-l);
+  position: relative;
+  display: flex;
+  display: -webkit-flex;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+}
+
+.ListItem--subitem {
+  padding: var(--space-m) var(--space-l);
+}
+
+.ListItem--miniitem {
+  padding: var(--space-s) var(--space-l);
+}
+
+.ListItem.is-active {
+  background-color: var(--color-alto);
+}
+
+.ListItem.is-clickable {
+  cursor: pointer;
+}
+
+.ListItem.is-clickable:hover {
+  background-color: var(--color-grey);
+}
+
+.ListItem-divider {
+  border-bottom: 1px solid var(--color-grey);
+}
+
+.ListItem-title {
+  color: var(--color-tolopea);
+  font-size: var(--font-size--m);
+  line-height: var(--line-height--m);
+}
+
+.ListItem-leftIcon {
+  width: var(--space-m);
+  margin: var(--space-m) 0;
+  margin-right: var(--space-xl);
+  display: flex;
+  display: -webkit-flex;
+  color: var(--color-tolopea);
+}
+
+.ListItem-subtitle {
+  color: var(--color-tolopea);
+  font-size: var(--font-size--s);
+  line-height: var(--line-height--m);
+  color: var(--color-granite);
+}
+
+.ListItem-valueContainer {
+  color: var(--color-tolopea);
+  font-size: var(--font-size--m);
+  line-height: var(--line-height--m);
+  margin-left: auto;
+  padding-left: var(--space-l);
+  display: flex;
+  display: -webkit-flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.ListItem-itemIcon {
+  width: var(--space-m);
+  margin: 0 var(--space-m);
+  display: flex;
+  display: -webkit-flex;
+  color: var(--color-tolopea);
+}
+
+@media only screen and (max-width: 460px) {
+  .ListItem {
+    padding: var(--space-xl) var(--space-l);
```

Comment:
How does it react with other sizes?

Reply:
If you're asking about another viewports - the specs required it only to change on a phone.

If you're asking about other sizes of `ListItem` component (normal, sub, mini) it's been changed now to:

```
.ListItem:not(.ListItem--miniitem):not(.ListItem--subitem) {
```

so that i'ts only applied to the normal item

Reply:
Alright 👍 
Let's hope this specificity doesn't bite us back 🙂